### DISCUSSION TITLE
Docker images of ComfyUI with FLUX support?

### DISCUSSION THREAD

**Discussion by Szaamaan** [comfyanonymous/ComfyUI#5999] [Q&A]
To keep things short - I'm looking for dockerized implementations of ComfyUI. I'm currently using `yanwk/comfyui-boot:cu121` and for the MOST part it seems to be sufficient. However I couldn't get it to work with a FLUX model / workflow and then it just crashes.

Any suggestions on a proper dockerized version that works with FLUX?

**Comment by LukeG89** [comment]
I don't use Docker, but I think this one should work (linked by @Amorano on Discord): https://github.com/ai-dock/comfyui
